quickwit/quickwit-metastore/src/tests.rs

```diff
@@ -2064,6 +2070,61 @@ pub mod test_suite {
                     "list-splits-six",
                 ])
             );
+
+            let query =
+                ListSplitsQuery::for_index(index_id).with_update_timestamp_gte(current_timestamp);
```

Comment:
Here we could run the test with `with_update_timestamp_gte(split_6_create_timestamp)` (just create the variable above when create the split 6), so we can check the filtering too.